Source/MSBuildProjectBuilder/Assets/MSBuildProjectBuilder/Editor/MSBuildProjectBuilderEditor.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace MSBuildForUnity
```

Comment:
Maybe `Microsoft.MSBuild` to better meet the [naming guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-namespaces) for namespaces?

Reply:
MSBuild itself uses the `Microsoft.Build` namespace. Maybe the namespace could be `Microsoft.Build.Unity`?

Reply:
I generally dislike putting the platform into the namespace, but I guess in this case it's also the Feature? I can't think of a better option so I'm down. =)

Reply:
I'd be fine with just `Microsoft.Build` too.